src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  padding: 10px 0;
+}
```

Comment:
Your header is flexbox, so it's better to align elements using `align-items` instead of padding. That way will allow us to don't worry about changes of logo size of header height (now it can be broken by this changes)